tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolBIRTest.java

```diff
@@ -90,6 +94,41 @@ public void setup() {
         srcFile = getDocumentForSingleSource(project);
     }
 
+    // Tests if the field symbols of a record type defined in testorg/testproject are accessible
+    @Test(dataProvider = "RecordFieldImports")
+    public void testRecordFieldSymbol(int line, int col, String fieldName, TypeDescKind typeKind, boolean isOptional,
+                                      boolean hasDefaultValue, String signature) {
+        Project project = BCompileUtil.loadProject("test-src/symbol_lookup_with_record_field_imports_test.bal");
+        Package currentPackage = project.currentPackage();
+        ModuleId defaultModuleId = currentPackage.getDefaultModule().moduleId();
+        Document srcFile = getDocumentForSingleSource(project);
+
+        PackageCompilation packageCompilation = currentPackage.getCompilation();
+        SemanticModel model = packageCompilation.getSemanticModel(defaultModuleId);
```

Comment:
There are a couple of util methods to simplify this. See the following for example: https://github.com/ballerina-platform/ballerina-lang/blob/f1515f27806ad0d3b239f852ad4994ec25c434ed/tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/ClassSymbolTest.java#L59

Reply:
Simplified.